sympy/utilities/lambdify.py

```diff
@@ -335,18 +416,22 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
         namespace.update(buf)
 
     if hasattr(expr, "atoms"):
-        #Try if you can extract symbols from the expression.
-        #Move on if expr.atoms in not implemented.
+        #   Try if you can extract symbols from the expression.
+        # Move on if expr.atoms in not implemented.
```

Comment:
Please keep all of these kinds of fixes in a separate PR (or at least separate commit). I makes it difficult to focus on the actual code changes.